packages/ag-charts-website/src/features/homepage/components/faqs/Faqs.tsx

```diff
@@ -16,44 +16,58 @@ export const Faqs: React.FC = () => {
+        'Pricing page': 'https://www.ag-grid.com/license-pricing/?utm_source=charts-homepage&utm_medium=faqs',
+        themes: 'https://www.ag-grid.com/charts/react/themes/?utm_source=charts-homepage&utm_medium=faqs',
+        'Theme Builder': 'https://www.ag-grid.com/theme-builder/?utm_source=charts-homepage&utm_medium=faqs',
+        documentation: 'https://www.ag-grid.com/charts/react/quick-start/?utm_source=charts-homepage&utm_medium=faqs',
+        tooltips: 'https://www.ag-grid.com/charts/react/tooltips/?utm_source=charts-homepage&utm_medium=faqs',
+        zooming: 'https://www.ag-grid.com/charts/react/zoom/?utm_source=charts-homepage&utm_medium=faqs',
+        panning: 'https://www.ag-grid.com/charts/react/navigator/?utm_source=charts-homepage&utm_medium=faqs',
+        'context menus':
+            'https://www.ag-grid.com/charts/react/context-menu/?utm_source=charts-homepage&utm_medium=faqs',
+        'click events': 'https://localhost:4600/charts/react/events/?utm_source=charts-homepage&utm_medium=faqs',
+        'export your charts': 'https://localhost:4600/charts/react/api-download',
+        'AG Charts React':
+            'https://www.ag-grid.com/charts/react/quick-start/?utm_source=charts-homepage&utm_medium=faqs',
+        accessibility: 'https://www.ag-grid.com/charts/react/accessibility/?utm_source=charts-homepage&utm_medium=faqs',
+        axes: 'https://www.ag-grid.com/charts/react/axes-types/?utm_source=charts-homepage&utm_medium=faqs',
+        'series types': 'https://www.ag-grid.com/charts/gallery/?utm_source=charts-homepage&utm_medium=faqs',
+        'real time':
+            'https://blog.ag-grid.com/updating-bar-chart-in-real-time/?utm_source=charts-homepage&utm_medium=faqs',
     };
 
     // Function to update FAQ json with links
     const createLinkedText = (text: string, linkMap: Record<string, string>) => {
         const elements: React.ReactNode[] = [];
-        let remainingText = text;
-        Object.keys(linkMap).forEach((term) => {
-            const url = linkMap[term];
-            const parts = remainingText.split(term);
-            if (parts.length > 1) {
-                elements.push(parts.shift());
+        const terms = Object.keys(linkMap);
+        if (terms.length === 0) return [text];
+
+        const pattern = new RegExp(`(${terms.map((term) => escapeRegExp(term)).join('|')})`, 'gi');
+        function escapeRegExp(string: string) {
+            return string.replace(/[.*+?^${}()|[\]\\]/g, '\\$&');
+        }
+
+        const parts = text.split(pattern);
+        parts.forEach((part, index) => {
+            if (terms.includes(part)) {
+                const url = linkMap[part];
                 elements.push(
-                    <a href={url} target="_blank" rel="noopener noreferrer" key={url}>
-                        {term}
+                    <a href={url} target="_blank" rel="noopener noreferrer" key={`${url}-${index}`}>
```

Comment:
It's generally not recommended to use `index` as part of the key, as the array order can change between react renders. `part` seems like it'd work here

```suggestion
                    <a href={url} target="_blank" rel="noopener noreferrer" key={`${url}-${part}`}>
```